b2share/modules/records/triggers.py

```diff
@@ -82,3 +98,17 @@ def index_record(record):
         version=record.revision_id,
         version_type='external_gte',
     )
+
+
+def check_record_immutable_fields(record):
```

Comment:
I'll see when the permissions commit is ready. If the change is not too big I'll include it.